packages/ifpack2/src/Ifpack2_LocalSparseTriangularSolver_decl.hpp

```diff
@@ -107,6 +107,9 @@ class LocalSparseTriangularSolver :
   //! Specialization of Tpetra::RowMatrix used by this class.
   typedef Tpetra::RowMatrix<scalar_type, local_ordinal_type,
                             global_ordinal_type, node_type> row_matrix_type;
+  //! Specialization of Tpetra::CrsMatrix used by this class.
+  typedef Tpetra::CrsMatrix<scalar_type, local_ordinal_type,
+                            global_ordinal_type, node_type, false> crs_matrix_type;
```

Comment:
@aprokop Ah, it's a forward declaration issue.  Nalu has this issue too.  Never mind then :-) This reminds me that Tpetra will need forward declaration headers at some point.